internal/pkg/service/cli/cmd/env.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/env"
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+)
+
+func loadEnvFiles(ctx context.Context, logger log.Logger, osEnvs *env.Map, fs filesystem.Fs) *env.Map {
+	// File system basePath = projectDir, so here we are using current/top level dir
+	projectDir := `.` // nolint
```

Comment:
Copied from the deprecated `options` package.

The `options` package should be removed at the end.